vllm/engine/llm_engine.py

```diff
@@ -14,6 +14,7 @@
 from vllm.transformers_utils.tokenizer import (detokenize_incrementally,
                                                get_tokenizer)
 from vllm.utils import Counter
+import copy
```

Comment:
Nit: maybe move this copy to the beginning of this file?

Reply:
ok, I updated the code